.github/ci.sh

```diff
@@ -102,6 +102,7 @@ bundle_files() {
   cp docs/*pdf $doc
   mkdir -p $lib
   cp -r lib/* $lib
+  cp -r syntax-highlight $lib
```

Comment:
This will put the syntax-highlighting–related files among the `.cry` files, which is perhaps not the most intuitive place to put them. I wonder if it would make more sense to have a separate `syntax-highlight/` subdirectory for this? Something like this, perhaps:

```
syntax_highlight=dist/share/syntax-highlight
...
mkdir -p $syntax_highlight
cp -r syntax-highlight $syntax_highlight
```

Reply:
Hm, I was hoping that since I don't have a slash after `syntax-highlight` it will copy the directory rather than the files in it... Did it not do that?

Reply:
It does copy the directory. What I am referring to is that you have a situation like this:

```
$ ls cryptol-2.13.0.99-ubuntu-22.04-x86_64/share/cryptol/
Array.cry         Cryptol.cry       Float.cry         SuiteB.cry        
Cryptol/          CryptolTC.z3      PrimeEC.cry       syntax-highlight/
```

Perhaps this is what you intended, but I'm not sure that I would have thought to look here for the `syntax-highlight` directory. I'll let you make the call.